### ISSUE TITLE
Ability to create a new machine

### ISSUE THREAD

**Issue by gkiely** [statelyai/xstate-tools#304]
It would be great if it were possible to create a new state chart using the VSCode extension, just like in the stately editor.

Current:
Xstate > Open Visual Editor in a blank file shows an error "Could not find a machine at the current cursor error".

Expected:
Ability to create a new machine in the visual editor.

Ideally via a separate command, and/or the visual editor shows a button to create a new state machine.

**Comment by Andarist** [comment]
Yeah, that makes sense and I was actually thinking about something similar recently. I was thinking about a command though and it seems that perhaps you'd like to have a button in the visual editor.

**Comment by gkiely** [comment]
Command works fine for me.

**Comment by davidkpiano** [comment]
You can also [use the `xsm` snippet](https://stately.ai/blog/xstate-vs-code-xsm-snippet) to easily create new machines in the code, although I agree, having a dedicated command would be a good experience.
